packages/react-scripts/scripts/init.js

```diff
@@ -172,10 +178,18 @@ module.exports = function(
     fs.unlinkSync(templateDependenciesPath);
   }
 
+  // yarn ws bug not creating app/node_modules/.bin link to hoisted react-scripts
```

Comment:
Can this be now removed considering `yarn` is at @1.6.0 and the fix landed in 1.5.1 in Feb (I think?)?

Of course there's the case that someone would run the script with a old version of `yarn` but 🤷‍♂️